pages/linux/nmcli-monitor.md

```diff
@@ -0,0 +1,8 @@
+# nmcli monitor
+
+> Monitor changes to the NetworkManager connection status.
+> More information: <https://developer.gnome.org/NetworkManager/stable/nmcli.html>.
+
+- Monitor NetworkManager changes:
```

Comment:
```suggestion
- Start monitoring NetworkManager changes:
```
What do you think?

Reply:
Doesn't really add anything IMO

Reply:
I think it just fits better, but if you prefer the original one I'm okay with that.

Reply:
Start monitoring is only used once in every page, so it seems outnumbered and out used.

Reply:
I like @marchersimon's suggestion better in this instance.